### DISCUSSION TITLE
Add "Spider_X" to reality Transmition

### DISCUSSION THREAD

**Discussion by farshadl** [Gozargah/Marzban#1152] [Q&A] [answered]
سلام.
توی آخرین نسخه امکان اضافه کردن پارامتر Spider_X به کانفیگ ریالیتی مهیا شده.
چطور باید Spider_X را به یک کانفیگ ریالیتی اضافه کرد؟ من این پارامتر را توی آخرین نسخه مرزبان ندیدم!
ممنون میشم @M03ED جواب من را بدید چون روی این قابلت شما کار کردید

**Comment by erfjab** [comment]
بهتر بود ایشوی فیچر کنید. دیسکاشن فضای برای صحبت و همفکری محسوب میشه. البته بازم ممکنه موسد عزیز اینجا جواب شمارو بدن

**Reply by farshadl** [reply]
ممنون. اگه جواب ندادن، اونجا مطرحش میکنم

**Comment by M03ED** [comment]
> سلام.
> توی آخرین نسخه امکان اضافه کردن پارامتر Spider_X به کانفیگ ریالیتی مهیا شده.
> چطور باید Spider_X را به یک کانفیگ ریالیتی اضافه کرد؟ من این پارامتر را توی آخرین نسخه مرزبان ندیدم!
> ممنون میشم @M03ED جواب من را بدید چون روی این قابلت شما کار کردید

صرفا مقدار رو به xray core اضافه کنید 
خودکار برای یوزر ارسال میشه ، دقت کنید که این پارامتر در سینگ باکس و کلش متا ساپورت نمیشه

**Reply by farshadl** [reply]
ممنون @M03ED 
من توی inbound ریالیتی، توی reality settings مقدار spiderX را به صورت زیر وارد کردم ولی بعد از آپدیت اشتراک، به v2rayNG اضافه نمیشه! اشتباهم کجاست؟

      "realitySettings": {
          "dest": "www.telewebion.com:443",
          "maxClient": "",
          "maxTimediff": 0,
          "minClient": "",
          "privateKey": "kBBUrKgBgs9SEzbs-igFKkt-B9L6H2EVpY7AWW38Hko",
          "serverNames": [
            "www.telewebion.com"
          ],
          "shortIds": [
            "2c3c9d3f"
          ],
          "spiderX": [
            "/dsfbr23"
          ],
          "show": false,
          "xver": 0
        },

**Reply by farshadl** [reply]
آقا موسد میشه لطفا بیشتر راهنمایی کنید که چطور میشه spiderX را اضافه کرد؟
@M03ED

**Reply by M03ED** [reply]
> ممنون @M03ED 
> من توی inbound ریالیتی، توی reality settings مقدار spiderX را به صورت زیر وارد کردم ولی بعد از آپدیت اشتراک، به v2rayNG اضافه نمیشه! اشتباهم کجاست؟
> 
>       "realitySettings": {
>           "dest": "www.telewebion.com:443",
>           "maxClient": "",
>           "maxTimediff": 0,
>           "minClient": "",
>           "privateKey": "kBBUrKgBgs9SEzbs-igFKkt-B9L6H2EVpY7AWW38Hko",
>           "serverNames": [
>             "www.telewebion.com"
>           ],
>           "shortIds": [
>             "2c3c9d3f"
>           ],
>           "spiderX": [
>             "/dsfbr23"
>           ],
>           "show": false,
>           "xver": 0
>         },

اشتباه این جاست که به صورت لیست وارد شده ولی spiderX فقط یک مقدار می‌تونه داشته باشه

**Reply by farshadl** [reply]
مقدار SpiderX را از حالت لیست خارج کردم ولی بازم بعد از آپدیت اشتراک، تو برنامه v2rayNG اضافه نشد. تنظیمات جدید را به صورت زیر وارد کردم:


  "realitySettings": {
      "dest": "www.telewebion.com:443",
      "maxClient": "",
      "maxTimediff": 0,
      "minClient": "",
      "privateKey": "kBBUrKgBgs9SEzbs-igFKkt-B9L6H2EVpY7AWW38Hko",
      "serverNames": [
        "www.telewebion.com"
      ],
      "shortIds": [
        "2c3c9d3f"
      ],
      "spiderX": "/dsfbr23",
      "show": false,
      "xver": 0
    },


باگی توی مرزبان وجود داره؟ یا اینکه مورد دیگه‌ای هست؟ 
آیا پارامتر spiderX باید توی inbound وارد بشه؟

**Comment by farshadl** [comment, marked as answer]
من متوجه شدم که کجا را اشتباه می‌کردم. توی تنظیمات inbound باید پارامتر را با حرف اول بزرگ نوشت. بصورت SpiderX. تنظیمات میشه مشابه زیر:

  "realitySettings": {
      "dest": "www.telewebion.com:443",
      "maxClient": "",
      "maxTimediff": 0,
      "minClient": "",
      "privateKey": "kBBUrKgBgs9SEzbs-igFKkt-B9L6H2EVpY7AWW38Hko",
      "serverNames": [
        "www.telewebion.com"
      ],
      "shortIds": [
        "2c3c9d3f"
      ],
      "SpiderX": "/dsfbr23",
      "show": false,
      "xver": 0
    },

**Reply by amiredi** [reply]
فعال کردن این گزینه چه کمکی میتونه بکنه؟
